services/blockchain-indexer/shared/indexer/tokenIndex.js

```diff
@@ -20,20 +20,30 @@ const {
 
 const { requestConnector } = require('../utils/request');
 
+const { isMainchain } = require('../common');
+
 const config = require('../../config');
 
 const topLSKAddressesIndexSchema = require('../database/schema/topLSKAddresses');
 
 const MYSQL_ENDPOINT = config.endpoints.mysql;
 
+let CHAIN_ID;
+const LOCAL_ID_LSK = '00000000';
+
 const getTopLSKAddressesIndex = () => getTableInstance(
 	topLSKAddressesIndexSchema.tableName,
 	topLSKAddressesIndexSchema,
 	MYSQL_ENDPOINT,
 );
 
 const getLiskBalanceByAddress = async (address) => {
-	const LISK_TOKEN_ID = config.tokens.lisk.id;
+	if (!CHAIN_ID) {
+		const status = await requestConnector('getNetworkStatus');
+		CHAIN_ID = status.chainID;
+	}
+
+	const LISK_TOKEN_ID = CHAIN_ID.concat(LOCAL_ID_LSK);
```

Comment:
Can we cache the concatenated value to avoid the string concatenation every single time the method is invoked?